api/resources/component.go

```diff
@@ -55,19 +58,53 @@ type Component struct {
 	Name                   string                 `json:"name"`
 }
 
+type CPUQuantity struct {
+	resource.Quantity
+}
+
+func (c *CPUQuantity) MarshalJSON() ([]byte, error) {
+	capInStr := strconv.FormatInt(c.MilliValue(), 10)
+	return []byte(fmt.Sprintf(`"%s"`, capInStr)), nil
```

Comment:
Why the string should include quotes?

Reply:
without the quote, the marshal result will be something like 100 instead of "100"